products/sqlite.md

```diff
@@ -0,0 +1,333 @@
+    eol: false
+    latest: "3.11.1"
+    latestReleaseDate: 2016-03-03
+    releaseDate: 2016-02-15
+
+-   releaseCycle: "3.10"
+    eol: false
+    latest: "3.10.2"
+    latestReleaseDate: 2016-01-20
+    releaseDate: 2016-01-06
+
+-   releaseCycle: "3.09"
+    eol: false
+    latest: "3.09.02"
+    latestReleaseDate: 2015-11-02
+    releaseDate: 2015-10-14
+
+-   releaseCycle: "3.08"
+    eol: false
+    latest: "3.08.11.01"
+    latestReleaseDate: 2015-07-29
+    releaseDate: 2013-08-26
+
+-   releaseCycle: "3.07"
+    eol: false
+    latest: "3.07.17"
+    latestReleaseDate: 2013-05-20
+    releaseDate: 2010-07-21
+
+-   releaseCycle: "3.06"
+    eol: false
+    latest: "3.06.23.01"
+    latestReleaseDate: 2010-03-26
+    releaseDate: 2008-07-16
+
+-   releaseCycle: "3.05"
+    eol: false
+    latest: "3.05.09"
+    latestReleaseDate: 2008-05-14
+    releaseDate: 2007-09-04
+
+-   releaseCycle: "3.04"
+    eol: false
+    latest: "3.04.02"
+    latestReleaseDate: 2007-08-13
+    releaseDate: 2007-06-18
+
+-   releaseCycle: "3.03"
+    eol: false
+    latest: "3.03.17"
+    latestReleaseDate: 2007-04-25
+    releaseDate: 2006-01-11
+
+-   releaseCycle: "3.02"
+    eol: false
+    latest: "3.02.08"
+    latestReleaseDate: 2005-12-19
+    releaseDate: 2005-03-21
+
+-   releaseCycle: "3.01"
+    eol: false
+    latest: "3.01.06"
+    latestReleaseDate: 2005-03-17
+    releaseDate: 2005-01-21
+
+-   releaseCycle: "3.0"
+    eol: false
+    latest: "3.0.8"
+    latestReleaseDate: 2004-10-12
+    releaseDate: 2004-06-18
+
+-   releaseCycle: "2.08"
+    eol: false
+    latest: "2.8.14"
+    latestReleaseDate: 2004-06-09
+    releaseDate: 2003-02-16
+
+-   releaseCycle: "2.07"
+    eol: false
+    latest: "2.7.6"
+    latestReleaseDate: 2003-01-25
+    releaseDate: 2002-08-25
+
+-   releaseCycle: "2.06"
+    eol: false
+    latest: "2.6.3"
+    latestReleaseDate: 2002-08-13
+    releaseDate: 2002-07-18
+
+-   releaseCycle: "2.05"
+    eol: false
+    latest: "2.5.6"
+    latestReleaseDate: 2002-07-07
+    releaseDate: 2002-06-17
+
+-   releaseCycle: "2.04"
+    eol: false
+    latest: "2.04.12"
+    latestReleaseDate: 2002-05-10
+    releaseDate: 2002-03-11
+
+-   releaseCycle: "2.03"
+    eol: false
+    latest: "2.03.03"
+    latestReleaseDate: 2002-02-19
+    releaseDate: 2002-02-03
+
+-   releaseCycle: "2.02"
+    eol: false
+    latest: "2.02.05"
+    latestReleaseDate: 2002-01-28
+    releaseDate: 2001-12-22
+
+-   releaseCycle: "2.01"
+    eol: false
+    latest: "2.01.07"
+    latestReleaseDate: 2001-12-15
+    releaseDate: 2001-11-12
+
+-   releaseCycle: "2.0"
+    eol: false
+    latest: "2.00.08"
+    latestReleaseDate: 2001-11-04
+    releaseDate: 2001-09-28
+
+-   releaseCycle: "1.0"
+    eol: false
+    latest: "1.00.32"
+    latestReleaseDate: 2001-07-23
+    releaseDate: 2000-08-17
+
+---
+
+> SQLite is an in-process library that implements a self-contained, serverless, zero-configuration, 
+> transactional SQL database engine.
+
```

Comment:
This empty line should be removed (or it will be considered two distinct citations).